db/fixtures/ae_datastore/ManageIQ/System/Event/EmsEvent/RHEVM.class/user_add_vm_template_finished_failure.yaml

```diff
@@ -7,6 +7,4 @@ object:
     name: USER_ADD_VM_TEMPLATE_FINISHED_FAILURE
     inherits: 
     description: 
-  fields:
-  - rel4:
-      value: "/System/event_handlers/event_action_policy?target=src_vm&policy_event=vm_discover&param="
+  fields: []
```

Comment:
Ultimately that event might be removed from the `event_groups` section of the yaml file but that has nothing to do with this PR.  The goal was to remove the vm_discover policy event.  We still capture the `USER_ADD_VM_TEMPLATE_FINISHED_FAILURE` event from the vendor and process it through automate like before we are just no raising the `vm_discover` policy event.